File: KungFury.cpp
#include <chrono>
#include <iostream>
#include <fstream>
#include <sstream>
#include <thread>

#include <cstdlib>
#include <ctime>

using namespace std;

int main() {
    ifstream file("script.txt");
    if (!file.is_open()) {
        cout << "Could not open file" << endl;
        exit(-1);
    }

    stringstream buffer;
    buffer << file.rdbuf();
    for (int i = 0; i < 2; i++) {
        for (auto& c : buffer.str()) {
            cout << c << flush;
            this_thread::sleep_for(chrono::milliseconds(20));
        }
    }

    int count = 0;
    const char* chars 
        = "0123456789" 
        "ABCDEFGHIJKLMNOPQRSTUVWXYZ" 
        "abcdefghijklmnopqrstuvwxyz";
    srand(time(nullptr));

    while (true) {
        cout << chars[rand() % 62] << flush;
        if (count % 80 == 0) {
            cout << endl;
        }

        this_thread::sleep_for(chrono::milliseconds(10));
        count++;
    }

    return 0;
}

File: kung_fury.go
package main

import (
    "fmt"
    "io/ioutil"
    "math/rand"
    "os"
    "time"
)

func main() {
    file, err := ioutil.ReadFile("script.txt")
    if err != nil {
        fmt.Println("Could not open file")
        os.Exit(-1)
    }

    energyConverterText := string(file)
    for i := 0; i < 2; i++ {
        for _, c := range energyConverterText {
            fmt.Print(string(c))
            time.Sleep(20 * time.Millisecond)
        }
    }

    count := 0
    chars := "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
    rand.Seed(time.Now().UTC().UnixNano())

    for {
        fmt.Print(string(chars[rand.Intn(len(chars))]))
        if count % 80 == 0 {
            fmt.Print("\n")
        }

        time.Sleep(10 * time.Millisecond)
        count++
    }
}
